### PR TITLE
[MapD] Added crossjoin operator

### LINKED ISSUE #1621: MapD histogram() : KeyError: <class 'ibis.expr.operations.CrossJoin'>
https://docs.ibis-project.org/notebooks/tutorial/8-More-Analytics-Helpers.html      
In [11]:
`t = con.table('functional_alltypes')

d = t.double_col

tier = d.histogram(10).name('hist_bin')
expr = (t.group_by(tier)
        .aggregate([d.min(), d.max(), t.count()])
        .sort_by('hist_bin'))
expr.execute()`

return error : 
  File "/home/sw/anaconda3/lib/python3.6/site-packages/ibis/sql/compiler.py", line 1808, in _get_join_type
    return self._join_names[type(op)]
KeyError: <class 'ibis.expr.operations.CrossJoin'>

Backend=MapD    MapD=4.1.1    ibis=0.14    Python=3.6
Ubuntu=18.04    Anaconda=5.1

### PR DESCRIPTION
**Pull request by xmnlab**
This PR fixes #1621

### CODE REVIEW

File: ibis/mapd/compiler.py [hunk 1]
```diff
@@ -150,6 +151,10 @@ def get_result(self):
                 fmt_preds = util.indent('USING ' + ', '.join(fmt_preds),
                                         self.indent * 2)
                 buf.write(fmt_preds)
+            else:
+                buf.write(
+                    util.indent('ON (TRUE)', self.indent * 2)
```
Comment (cpcloud) [inline ibis/mapd/compiler.py]: Does this need the parentheses?

File: ibis/mapd/operations.py [hunk 2]
```diff
@@ -324,6 +324,13 @@ def _set_literal_format(translator, expr):
     return '({})'.format(', '.join(formatted))
 
 
+def _cross_join(translator, expr):
+    args = expr.op().args
+    t1, t2 = args[:2]
```
Comment (cpcloud) [inline ibis/mapd/operations.py]: I think these have names, like `left` and `right`. Can you use those if they exist?

File: ibis/mapd/operations.py [hunk 3]
```diff
@@ -324,6 +324,13 @@ def _set_literal_format(translator, expr):
...
+
```
Comment (cpcloud) [inline ibis/mapd/operations.py]: Kill this newline.

File: ibis/mapd/compiler.py (hunk 1 above)
Comment (xmnlab) [reply ibis/mapd/compiler.py]: I think it is not necessary ... thanks!

File: ibis/mapd/operations.py (hunk 2 above)
Comment (xmnlab) [reply ibis/mapd/operations.py]: You're right! the names are left and right. thanks!

File: ibis/mapd/operations.py (hunk 3 above)
Comment (xmnlab) [reply ibis/mapd/operations.py]: done!

**Comment by xmnlab** [comment]
@cpcloud the conda build is failing for windows. do you have any idea about this error?

**Comment by xmnlab** [comment]
same problem with mariadb on windows test py35

**Comment by cpcloud** [comment]
Thanks @xmnlab !

**Pull request closed without merging**